Japx/Classes/Core/Japx.swift

```diff
@@ -266,13 +331,20 @@ private extension Japx.Decoder {
 
 private extension Japx.Decoder {
  
-    private static func resolve(object: Parameters, allObjects: [TypeIdPair: Parameters], paramsDict: NSDictionary) throws -> Parameters {
+    static func resolve(object: Parameters, allObjects: [TypeIdPair: Parameters], paramsDict: NSDictionary, options: JapxDecodingOptions) throws -> Parameters {
         var attributes = (try? object.dictionary(for: Consts.APIKeys.attributes)) ?? Parameters()
         attributes[Consts.APIKeys.type] = object[Consts.APIKeys.type]
         attributes[Consts.APIKeys.id] = object[Consts.APIKeys.id]
         
         let relationshipsReferences = object.asDictionary(from: Consts.APIKeys.relationships) ?? Parameters()
         
+        //Force cast here is ok since it is taken from include list string
```

Comment:
Which force cast? 😄 also space between // and comment

Reply:
🤦‍♂ Comment few lines to high.